samples/client/petstore/kotlin/src/main/kotlin/org/openapitools/client/models/User.kt

```diff
@@ -17,6 +17,7 @@ package org.openapitools.client.models
 
 
 import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
```

Comment:
Is there a way to only add this import when an enum class is inside the data class?

Reply:
Not that I'm aware of